### DISCUSSION TITLE
docker.io/kadalu/kadalu-operator:devel in release ?!

### DISCUSSION THREAD

**Discussion by jkonecny75** [kadalu/kadalu#944] [Q&A] [answered]
Is it a mistake ? https://github.com/kadalu/kadalu/blob/1.0.0/manifests/kadalu-operator-rke.yaml#L394

**Comment by vatsa287** [comment, marked as answer]
This is from devel branch.
For 1.0 release or any other tag please check https://github.com/kadalu/kadalu/releases yaml files.

**Reply by jkonecny75** [reply]
OK, that explains every our errors in logs (I used YAML files from branch instead from release) Now it is clear. Thank you

**Reply by vatsa287** [reply]
Welcome @jkonecny75 !
